pyprof/prof/pointwise.py

```diff
@@ -145,13 +149,13 @@ def bytes_flops(self):
 
         # Unary
         if self.op() in Pointwise.unary + Pointwise.representation:
-            assert (len(self.input) == 1)
+            assert (len(self.input) >= 1)
```

Comment:
Reason for weakening the assertion?

Reply:
Also - related to arg filtering problem.  As soon as args have a meaningful name the filtering breaks down.

Reply:
The problem is hacking / overloading an existing keyword (which identifies args and kwargs) for something else.